specification/servicelinker/ServiceLinker.Management/DryrunResource.tsp

```diff
@@ -0,0 +1,142 @@
+  ...ResourceNameParameter<
+    Resource = DryrunResource,
+    KeyName = "dryrunName",
+    SegmentName = "dryruns",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface DryrunResourceOps
+  extends Azure.ResourceManager.Legacy.LegacyOperations<
+      {
+        ...ApiVersionParameter,
+        ...SubscriptionIdParameter,
+        ...ResourceGroupParameter,
+        ...Azure.ResourceManager.Legacy.Provider,
+        ...LocationParameter,
+      },
+      KeysOf<ResourceNameParameter<
+        Resource = DryrunResource,
+        KeyName = "dryrunName",
+        SegmentName = "dryruns",
+        NamePattern = ""
+      >>
+    > {}
+
+@armResourceOperations
+interface DryrunResources {
+  /**
+   * get a dryrun job
+   */
+  getDryrun is DryrunResourceOps.Read<DryrunResource>;
+
+  /**
+   * create a dryrun job to do necessary check before actual creation
+   */
+  createDryrun is DryrunResourceOps.CreateOrUpdateAsync<DryrunResource>;
+
+  /**
+   * update a dryrun job to do necessary check before actual creation
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/lro-location-header" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @patch(#{ implicitOptionality: false })
+  updateDryrun is DryrunResourceOps.CustomPatchAsync<
+    DryrunResource,
+    PatchModel = DryrunPatch,
+    Response = ArmResourceUpdatedResponse<DryrunResource> | ArmAcceptedLroResponse<LroHeaders = ArmAsyncOperationHeader<FinalResult = DryrunResource>>
+  >;
+
+  /**
+   * delete a dryrun job
+   */
+  deleteDryrun is DryrunResourceOps.DeleteSync<DryrunResource>;
+
+  /**
+   * list dryrun jobs
+   */
+  listDryrun is DryrunResourceOps.List<
+    DryrunResource,
+    Response = ArmResponse<DryrunList>
+  >;
+}
+
+@armResourceOperations
+interface LinkersOps
+  extends Azure.ResourceManager.Legacy.LegacyOperations<
+      {
+        ...ApiVersionParameter,
+
+        #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+        @path(#{ allowReserved: true })
+        resourceUri: string,
+
+        ...Azure.ResourceManager.Legacy.Provider,
+      },
+      KeysOf<ResourceNameParameter<
+        Resource = DryrunResource,
+        KeyName = "dryrunName",
+        SegmentName = "dryruns",
+        NamePattern = ""
+      >>
+    > {}
+@armResourceOperations
+interface Linkers {
+  /**
+   * get a dryrun job
+   */
+  getDryrun is LinkersOps.Read<DryrunResource>;
+
+  /**
+   * create a dryrun job to do necessary check before actual creation
+   */
+  createDryrun is LinkersOps.CreateOrUpdateAsync<DryrunResource>;
+
+  /**
+   * add a dryrun job to do necessary check before actual creation
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/lro-location-header" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
Is this an lro? And if so, how does it perform polling?

Reply:
Yes, LRO! I didn't get your question actually. How service perform polling? I feel like it's not a question to me.